### PR TITLE
Styx: workaround for socket.timeout and faster shutdown

### PR DESCRIPTION
**Pull request by avrabe**
A wrong implementation in eventlet let the server die
after some time if there was no initial connection.
It was not solved yet (https://github.com/eventlet/eventlet/issues/222).
Retrigger the server after this exception.

Ros and eventlet's are not playing too nice together
when it is time to shutdown the service.
Trigger a sigkill to enforce this.

Few pylint cleanups.

Signed-off-by: Ralf Anton Beier <ralf_beier@me.com>

### CODE REVIEW

**Comment by avrabe** [comment]
Unfortunately I don't see a simple way out of the eventlet loop, except for killing the service.

File: ros/src/styx/bridge.py [hunk 1]
```diff
@@ -79,7 +77,8 @@ def create_pose(self, x, y, z, yaw=0.):
         pose.pose.position.y = y
         pose.pose.position.z = z
 
-        q = tf.transformations.quaternion_from_euler(0., 0., math.pi * yaw/180.)
+        q = tf.transformations.quaternion_from_euler(
```
Comment (asimonov) [inline ros/src/styx/bridge.py:80]: what is the line length that you target? 80 chars?

**Review by asimonov — APPROVED** [review]
looks great. i was wondering why the whole thing waits when I kill it. this explains it :)

File: ros/src/styx/bridge.py (hunk 1 above)
Comment (avrabe) [reply ros/src/styx/bridge.py:80]: Usually I aim for 120. I think I forgot to mention the --max-linenum while doing the autopep8.

**Comment by asimonov** [comment]
120 would be great! 80 is too limiting, in my opinion.

**Pull request merged**